src/components/Modal/BaseModal.tsx

```diff
@@ -110,13 +98,10 @@ function BaseModal({
         if (shouldSetModalVisibility) {
             Modal.setModalVisibility(true);
         }
-        onModalShow();
+        onModalShow?.();
```

Comment:
You defined default prop as an empty fucntion - in taht case conditional calling is needed?

Reply:
Updated